rental.ru

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book
+
```

Comment:
- [ ] Since a rental belongs to a book and also a person, Getters and Setters for @book and @person instance variables should be present